### PR TITLE
Clarify double-spend state

### LINKED ISSUE #446: Binding requests to connections
> For example, double spend state for unique, per-request redemption contexts only needs to exist within the scope of the request connection or session. 

This is probably not a good idea.  Clients will expect to be able to make a new request over a new connection and will probably not request fresh issuance if a request with a token fails.

(Also, "request connection" is not a concept in HTTP.)

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #446

### CODE REVIEW

File: draft-ietf-privacypass-auth-scheme.md [hunk 1]
```diff
@@ -302,14 +302,13 @@ This list is not exhaustive.
...
 Preventing double spending on tokens requires the origin to keep state
-associated with the redemption context. The size of this state varies based on
-the size of the redemption context. For example, double spend state for unique,
-per-request redemption contexts only needs to exist within the scope of
-the request connection or session. In contrast, double spend state for empty
-redemption contexts must be stored and shared across all requests until
-token-key expiration or rotation.
+associated with the redemption context. An empty redemption context is not
+bound to any property of the client request, so state to prevent double spending
+needs to be stored and shared across all servers that can accept request until
```
Comment (chris-wood) [inline draft-ietf-privacypass-auth-scheme.md:308]:
```suggestion
needs to be stored and shared across all origin servers that can verify and accept requests carrying tokens until
```

File: draft-ietf-privacypass-auth-scheme.md [hunk 2]
```diff
@@ -302,14 +302,13 @@ This list is not exhaustive.
...
+token-key expiration or rotation. For a non-empty redemption context, the
+double spend state only needs to be stored across the set of servers that will
```
Comment (chris-wood) [inline draft-ietf-privacypass-auth-scheme.md:310]:
```suggestion
double spend state only needs to be stored across the set of origin servers that will
```

**Comment by tfpauly** [comment]
The point here is to remove the reference to the "request connection", and just say "you need to keep the double spend state along with your redemption context", whatever that may be.

**Comment by chris-wood** [comment]
Right, but I don't think that resolves the linked issue.

**Pull request merged**